src/modules/helpers/eval.ts

```diff
@@ -81,7 +81,7 @@ export function fakeEval(
   do {
     let index: number;
     if (remaining.startsWith('(')) {
-      [index, current] = evalProcessFunction(remaining, current, expression);
+      [index, current] = evalProcessFunction(remaining, current);
     } else {
       [index, current] = evalProcessExpression(remaining, current);
     }
```

Comment:
These statements are now equal. The `if` statement is no longer required.


```suggestion
let [index, current] = evalProcessFunction(remaining, current);
```

Reply:
I'm stupid and can't read. These are not the same...

Reply:
Don't mind, I thought exactly the same when I had a first look 😆